components/screens/ManageDeliveries/ManageDelivery.tsx

```diff
@@ -170,6 +172,8 @@ class ManageDelivery extends React.Component<Props, State> {
         deliveryPlace.id === deliveryData.delivery.deliveryPlaceId
       );
 
+    await this.fetchContracts(deliveryData.product);
```

Comment:
Maybe should rename this method to fetchContractQuantities or something like that, currently it's bit confusing.